orchagent/aclorch.cpp

```diff
@@ -3436,6 +3565,15 @@ void AclOrch::doAclTableTask(Consumer &consumer)
                 }
             }
 
+            auto tableType = getAclTableType(tableTypeName);
+            if (!tableType)
+            {
+                it++;
+                continue;
```

Comment:
@bingwang-ms Yes, I believe this is the way to handle out of order tasks when there is a dependency.

Reply:
Yes, and I think there is similar issue with mirror session and acl rule. We will retry to create the acl rule until the referenced mirror session is not created. But I haven't got a solution.
Any idea what will happen if this happens?

Reply:
If I create a table that references unknown table type it will continue to retry till this unknown table type is created in config db or this table is deleted. Also, "warm-reboot" will not allow to proceed with such configuration. Unfortunatelly a simila issue exists with many other objects in orchagent (port, lags, vlans) and orchagent will retry untill a missing dependency is created.

Reply:
I see. Thanks.